src/javascript/Edit/save/save.request.js

```diff
@@ -25,19 +26,31 @@ export const saveNode = ({
     const dataToMutate = getDataToMutate({nodeData, formValues: values, sections, lang: language});
     const {childrenOrder, shouldModifyChildren} = getChildrenOrder(values, nodeData);
     const wipInfo = values[Constants.wip.fieldName];
+    let variables = adaptSaveRequest(nodeData, {
+        uuid: nodeData.uuid,
+        propertiesToSave: dataToMutate.propsToSave,
+        propertiesToDelete: dataToMutate.propsToDelete,
+        mixinsToAdd: dataToMutate.mixinsToAdd,
+        mixinsToDelete: dataToMutate.mixinsToDelete,
+        language,
+        shouldModifyChildren,
+        childrenOrder,
+        wipInfo
+    });
+
+    // Hooks on content to be created
+    const onEdits = registry.find({type: 'contentEditor.onEdit'});
+    variables = onEdits?.reduce((updatedVariables, registeredOnEdit) => {
+        try {
+            return registeredOnEdit.onCreate(updatedVariables, nodeData) || updatedVariables;
```

Comment:
It would be better to name the function onEdit instead of onCreate

Reply:
Thanks a lot for the comment, name updated!